operations/security_and_authentication/Add_LDAP_User_Federation.md

```diff
@@ -48,9 +50,64 @@ LDAP user federation is not currently configured in Keycloak. For example, if it
    
 2. Update the LDAP settings in the customizations.yaml file.
 
-   1. (Optional) Add the LDAP CA certificate in the certs.jks section of customizations.yaml.
+   1. (Optional) Repopulate the keycloak_users_localize and cray-keycloak Sealed Secrets in the customizations.yaml file with the desired configuration.
+      
+      Check to see if the `generate:` sections of the Sealed Secrets have been populated with encrypted Sealed Secrets already:
+
+      ```bash
+      ncn-m001# yq write -i ./customizations.yaml 'spec.kubernetes.sealed_secrets.keycloak_users_localize.generate.data.(args.name==ldap_connection_url).args.value' "ldap://$LDAP"
+      ncn-m001# yq read ./customizations.yaml spec.kubernetes.sealed_secrets.keycloak_users_localize
+      apiVersion: bitnami.com/v1alpha1
+      kind: SealedSecret
+      metadata:
+        annotations:
+          sealedsecrets.bitnami.com/cluster-wide: "true"
+        creationTimestamp: null
+        name: keycloak-users-localize
+      spec:
+        encryptedData:
+          ldap_connection_url: AgAmdO19GaLs3Yr9apnJ/JDuQS+6yMC+LlZrPO8g+9UvF2+0X1TifH/bPb0Dw4VMN/2MURx/vvwJE2DTz9yajuW3YJEdwD4o6z/OZ/qLDxu2u+HZSwRnWLWK6ROTBGMP7r0zOdQIDoeeAZw03+d4/UmiBlTlJhl+DJzcS8VbfJV+neNcZ0p5zcM7skqI5NL4teNItHoeuITC2QQ+TRQc/XOrkj3JxvrzFEtEstJz8fXOUXBOwakhRRzUZl9aAYcT6raK3mPQDg14AkM4JVCeku+h6O4OOoOIygC8FzfrXy+LWE93UZjw/0ZM+c6bqOLd7odGto5EylLaV7HS9V8trUPfKExBbYqoRzm+IU9eG3k8Gr7ijT9hRhr1wiV73DFy5NnNB0uAjFClnPXbqntbnSScLxHgFqrnitKM19RzcuHFaZ0Hq0S0VPO8az8BL7jiCkhlxqT0WN6I5RerHPt0PocikKJ/S58a2dc8uGwMgyAPCcJwNXnh7qoA3pgL72kD292ReCYUz6XR51XQAaW1S5O/OaI3VKCCHZAw+qCgW3tCraL3mjzuTYSQQQsDFicxcgQZVz26S/9ATW+3g8btOZlsJ8aA9zpSGAf+uo2N/OKDCmU60fxFTvddcYDlWZ9ZXM1q9lnDTzz1T+QMCZ/f+c2El4CgNvrKJvLueduuagGNfrYoVgGoQ76e/WVi866d1a0=
+        template:
+          metadata:
+            annotations:
+              sealedsecrets.bitnami.com/cluster-wide: "true"
+            creationTimestamp: null
+            name: keycloak-users-localize
+      generate:
+        data: {}
+      ```
+
+      Remove the existing keycloak_users_localize and cray-keycloak Sealed Secrets from customizations.yaml, and then add the `generate:` sections back in, populated with the desired configuration.
+
+      For example:
 
-      Follow step 8 under the "Create Baseline System Customizations" header in the [Prepare Site Init](../../install/prepare_site_init.md) procedure.
+      ```bash
+            cray-keycloak:
+                generate:
+                  name: keycloak-certs
+                  data:
+                    - type: static_b64
+                      args:
+                        name: certs.jks
+                        value: /u3+7QAAAAIAAAAA5yXvSDt11bGXyBA9M2iy0/5i1Tg=
```

Comment:
We seem to have lost the instructions for how to generate this value. The instructions to do that are in the "Create Baseline System Customizations" header in the [Prepare Site Init](../../install/prepare_site_init.md) procedure so that'll have to be copied in here, too.

It's the section "If LDAP requires TLS (recommended), update the cray-keycloak sealed secret value by supplying a base64 encoded Java ..."
and goes through "Then, inject and encrypt certs.jks.b64 into customizations.yaml:"